metron-interface/metron-alerts/src/app/service/alert.service.ts

```diff
@@ -39,21 +41,31 @@ export class AlertService {
               private ngZone: NgZone) { }
 
   public search(searchRequest: SearchRequest): Observable<AlertsSearchResponse> {
```

Comment:
Is it possible to make this service class match the rest layer?

Reply:
Done